script.js

```diff
@@ -1,12 +1,164 @@
-//You can edit ALL of the code here
 function setup() {
+  // this function loads the game of thrones episodes
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+// get all shows from shows.js
+const allShows = getAllShows();
+// Sorting the array of objects by shows' name
+allShows.sort(function (a, b) {
+  let nameA = a.name.toLowerCase();
+  let nameB = b.name.toLowerCase();
+  if (nameA < nameB) return -1;
+  if (nameA > nameB) return 1;
+  return 0;
+});
+// build the shows dropdown list
+allShows.forEach((show) => {
```

Comment:
as mentioned in the call, you should add this inside a function and once you do that, you can place ``showsDropDownElem`` inside the function (outside of the ``forEach`` loop) so that you don't need to keep getting the element